File: 351002/Ilian_Bukhovets/rest/internal/model/issue.go
package model

import "time"

type Issue struct {
	ID       int64     `db:"id" json:"id"`
	AuthorID int64     `db:"author_id" json:"authorId"`
	Title    string    `db:"title" json:"title"`
	Markers  []string  `db:"name" json:"markers"`
	Content  string    `db:"content" json:"content"`
	Created  time.Time `db:"created" json:"created"`
	Modified time.Time `db:"modified" json:"modified"`
}

File: 351003/Zarenok/demo/publisher/src/main/java/com/example/demo/model/Issue.java
package com.example.demo.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import jakarta.persistence.Id;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

import java.time.ZonedDateTime;
import java.util.List;

@Entity
@Setter
@Getter
@Table(name = "tbl_issue", schema = "dictcomp")
public class Issue {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "title", nullable = false, length = 64, unique = true)
    private String title;

    @Column(name = "content", unique = true)
    private String content;

    @CreationTimestamp
    @Column(name = "created", nullable = false)
    private ZonedDateTime created;

    @UpdateTimestamp
    @Column(name = "modified", nullable = false)
    private ZonedDateTime modified;

    @ManyToOne(optional = false)
    @JoinColumn(name = "author_id", nullable = false)
    private Author author;


    @ManyToMany
    @JoinTable(
            name = "tbl_issue_mark",
            joinColumns = @JoinColumn(name = "issue_id"),
            inverseJoinColumns = @JoinColumn(name = "mark_id")
    )
    private List<Mark> marks;
}
